From 5e8f9d884c985b608cb7e7d83b4ab204a9924e09 Mon Sep 17 00:00:00 2001
From: Andy Duan <fugang.duan@nxp.com>
Date: Mon, 9 Jan 2017 17:12:56 +0800
Subject: [PATCH 1471/5242] MLK-13735 ARM: dtsi: imx7ulp: set the lpuart
 module clock to 48Mhz

commit  c842b3187c83fc136608a180502e04d17f571ae0 from
https://source.codeaurora.org/external/imx/linux-imx.git

The untrimmed chip firc clock is 50Mhz after manually tuning.
Now the trimmed chip firc clock is stable to 48Mhz, so change
the lpuart module clock rate to 48Mhz.

Signed-off-by: Fugang Duan <fugang.duan@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7ulp.dtsi |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx7ulp.dtsi b/arch/arm/boot/dts/imx7ulp.dtsi
index a745a92..577c891 100644
--- a/arch/arm/boot/dts/imx7ulp.dtsi
+++ b/arch/arm/boot/dts/imx7ulp.dtsi
@@ -435,7 +435,7 @@
 			clock-names = "ipg";
 			assigned-clocks = <&clks IMX7ULP_CLK_LPUART7>;
 			assigned-clock-parents = <&clks IMX7ULP_CLK_FIRC>;
-			assigned-clock-rates = <50000000>;
+			assigned-clock-rates = <48000000>;
 			dmas = <&edma0 0 24>, <&edma0 0 23>;
 			dma-names = "tx","rx";
 			status = "disabled";
-- 
1.7.9.5

